vismne/utils/system.py

```diff
@@ -0,0 +1,8 @@
+""""""
```

Comment:
add link to original code + name of authors

Reply:
Done